compiler/rustc_typeck/src/check/wfcheck.rs

```diff
@@ -1479,5 +1479,12 @@ fn error_392(tcx: TyCtxt<'_>, span: Span, param_name: Symbol) -> DiagnosticBuild
     let mut err =
         struct_span_err!(tcx.sess, span, E0392, "parameter `{}` is never used", param_name);
     err.span_label(span, "unused parameter");
+    err.span_help(
```

Comment:
considering that the label above already uses the same span, maybe just using `help` is better here :thinking: 

It seems a bit verbose rn.

You might have to add this method using the `forward!` macro to `DiagnosticBuilder`